components/create_comment/create_comment.tsx

```diff
@@ -301,7 +334,7 @@ class CreateComment extends React.PureComponent {
     }
 
     componentWillUnmount() {
-        this.props.resetCreatePostRequest();
+        this.props.resetCreatePostRequest?.();
```

Comment:
When is this ever undefined?

Reply:
After a second look, apparently never. Probably wasn't marked as required on the PropTypes, and that is why I left it like optional.